DataSetExplorer/Program.cs

```diff
@@ -1,26 +1,254 @@
-﻿using DataSetExplorer.RepositoryAdapters;
+﻿using DataSetExplorer.DataSetBuilder.Model;
+using DataSetExplorer.RepositoryAdapters;
+using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.IO;
 
 namespace DataSetExplorer
 {
     class Program
     {
+        private static IDataSetAnalyzer _dataSetAnalysisService = new DataSetAnalysisService();
+        
         static void Main(string[] args)
         {
-            //IDataSetCreator dataSetCreationService = new DataSetCreationService("C:\\main-repository\\", new GitCodeRepository());
-            //dataSetCreationService.CreateDataSetSpreadsheet("MyProject", "https://github.com/wieslawsoltes/Core2D/tree/50ea0849bef7e211500764ffcd1fece4a3cb224e");
+            string chosenOption;
```

Comment:
Extract into ConsoleUI and use Program for DI. I've discussed this a bit more in my other review.